module-system/module-implementations/sov-bank/src/token.rs

```diff
@@ -107,6 +149,8 @@ impl<C: sov_modules_api::Context> Token<C> {
             name: token_name.to_owned(),
             total_supply,
             balances,
+            frozen,
+            authorized_minters: authorized_minters.unwrap_or(vec![C::Address::try_from(sender)?]),
```

Comment:
Here we put the sender of the tx as a minter only when the vec is empty. I think it would be more consistent to always insert it always (or never).

Reply:
ah ok! so the sender/creator is always part of the authorized minter. yeah, that was one of the questions i had. I can make that change